numpyro/contrib/nn/auto_reg_nn.py

```diff
@@ -58,7 +58,8 @@ def create_mask(input_dim, hidden_dims, permutation, output_dim_multiplier):
     return masks, mask_skip
 
 
-class AutoregressiveNN(object):
+def AutoregressiveNN(input_dim, hidden_dims, param_dims=[1, 1], permutation=None,
+                     skip_connections=False, nonlinearity=stax.Relu):
```

Comment:
todo: remove relu in distributions.util
todo: update autoiaf to incorporate changes in arn